assets/js/blocks/active-filters/block.tsx

```diff
@@ -34,7 +33,7 @@ import {
 } from './utils';
 import ActiveAttributeFilters from './active-attribute-filters';
 import FilterPlaceholders from './filter-placeholders';
-import { Attributes } from './types';
+import { Attributes, IsEditor } from './types';
```

Comment:
Instead of `import { Attributes, IsEditor } from './types';`, I think we should use `import type { ActiveFiltersBlockProps } from './types';`. This also means that we need to adjust `assets/js/blocks/active-filters/types.ts` accordingly. In addition, we should double-check other files within `assets/js/blocks/active-filters/`, to see if any of these files use `import { Attributes } from './types';`. If so, we should adjust the import and use `ActiveFiltersBlockProps` instead of `Attributes`.

Reply:
I checked and `index.ts` uses only `attributes` so I've left `Attributes` type as it is and shifted `isEditor` inside `ActiveFilterBlockProps` and exported it from types.ts into block.tsx and in index.tsx I have imported attributes only 
 does that sound good?